### PR TITLE
remove student, educator entities; add user

### PR DESCRIPTION
**Pull request by dem14n**

### CODE REVIEW

**Review by VitaliiZZzz — APPROVED** [review]
all bad, approved!

File: src/main/java/org/alvheim/sphinx/entities/User.java [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
 
-@Data
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
 @Entity
-@Table(name = "student")
-public class Student {
+@Table(name = "usr")
```
Comment (ForeverProglamer) [inline src/main/java/org/alvheim/sphinx/entities/User.java:23]: Maybe user instead of usr?)

Comment (dem14n) [reply src/main/java/org/alvheim/sphinx/entities/User.java:23]: no, postgresql doesn't allows anyone to use word "user". We wouldn't be able to create a relation with this name

Comment (ForeverProglamer) [reply src/main/java/org/alvheim/sphinx/entities/User.java:23]: got it

**Pull request merged**